frontend/src/components/PresentationElementTop/PresentationElementTop.jsx

```diff
@@ -1,5 +1,108 @@
 
 export default function PresentationElementTop() {
-  return <Style>PresentationElementTop</Style>;
+  return (
+    <Style>
+      <>
+        <h1 className="titlePresentationElementTop">Qui sommes-nous ?</h1>
+        <div className="presentationElementTopContainer">
+          <div className="presentationElementTop1">
+            <div className="imagebloc1">
+              <img src={Valuesimg} alt="Nos valeurs" className="Valuesimg" />
+            </div>
+            <div className="textbloc1">
+              <h2>Nos valeurs</h2>
+              <p>
+                Nous sommes avant tout une équipe d’experts IT, tous animés par
+                la même passion des relations humaines. L’intelligence
+                émotionnelle et l’éducation cognitive ne peuvent être remplacées
+                par des algorithmes. Notre cabinet de recrutement s’appuie sur
+                des méthodes authentiques, où l’humain est tout simplement
+                indispensable. Depuis 10 ans, externatic a développé un
+                savoir-faire sur le recrutement de profils pénuriques. Ces
+                compétences nous permettent d’intervenir sur d’autres secteurs
+                que l’IT. Notamment l’industrie avec <em>Induseo</em> et la
+                cybersécurité avec <em>Underguard</em>.
+              </p>
+            </div>
+          </div>
+          <div className="presentationElementTop2">
+            <div className="imagebloc2">
+              <img
+                src={Missionimg}
+                alt="Notre mission"
+                className="Missionimg"
+              />
+            </div>
+            <div className="textbloc2">
+              <h2>Notre mission</h2>
+              <p>
+                Notre mission de connecter les bonnes personnes, commence avec
+                des rencontres. Notre professionnalisme nous permet de
+                construire de vraies relations. Nous mettons en place des
+                équipes qui représentent les valeurs des entreprises sur le
+                long-terme. Notre vocation est de trouver les opportunités
+                professionnelles répondant aux “spécifications” de recherche de
+                nos candidats. Nous nous positionnons donc comme accompagnateur
+                du projet professionnel voire “agents de talents”.
+              </p>
+            </div>
+          </div>
+          <div className="presentationElementTop3">
+            <div className="imagebloc3">
+              <img
+                src={Engagementsimg}
+                alt="Nos engagements"
+                className="Engagementsimg"
+              />
+            </div>
+            <div className="textbloc3">
+              <h2 className="titlebloc3">Nos engagements</h2>
+              <p>
+                - Proposer des postes techniques uniquement auprès des startups,
+                éditeurs, DSI, etc. (pas d'ESN).
+                <br />
+                <br />
```

Comment:
Hmm, these tags are useless, it's CSS that should set the appearance:
- use `<li>`
- set the `inline-height` property